### DISCUSSION TITLE
KSP not working says im running on privileged account

### DISCUSSION THREAD

**Discussion by bkruz** [KSP-ModularManagement/KSPe#75] [General]
I'm launching on steam my game is fully updated and don't know where to go from here I don't know if these files will get you anywhere but that's what is in my logs folder 
[MMPatch.log](https://github.com/user-attachments/files/16937864/MMPatch.log)
[ModuleManager.log](https://github.com/user-attachments/files/16937865/ModuleManager.log)

@Lisias

**Comment by Lisias** [comment]
Hi, @bkruz . 

The Module Manager logs are not useful, I need the `KSP.log`. The information I need is there.

Are you running under Windows? Had you deactivated the UAC?

**Reply by Lisias** [reply]
Oh, in time!

Please run KSP until the problem happens, **then** quit KSP, and only then copy the `KSP.log`. Trying to copy the `KSP.log` with KSP still running usually truncates the log, removing important information.

**Comment by Lisias** [comment]
User abandoned the ticket. closing it.

**Comment by Lisias** [comment]
I'm necroing this issue to pinpoint to:

https://github.com/KSP-ModularManagement/KSPe/issues/56#issuecomment-5004682193

MS is going to deprecate UAC on Windows 11. (oh, boy)
